app/src/main/java/com/example/places/MainActivity.kt

```diff
@@ -13,6 +15,11 @@ class MainActivity : AppCompatActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
+        installSplashScreen().apply {
+            setKeepVisibleCondition{
+                viewModel.isLoaded.value
```

Comment:
Can't you make the condition follow whether the places livedata is empty or null? Instead of using a new tracker like isLoaded